apisix/plugins/response-rewrite.lua

```diff
@@ -115,6 +155,19 @@ function _M.check_schema(conf)
         end
     end
 
+    if conf.filters then
+        for _, filter in ipairs(conf.filters) do
+            if filter.regex == "" then
```

Comment:
It seems not good to skip this test, although `required` on `filter.regex`, but it can also fill with empty string `""`.
If could place non-empty rule, so we can skip this test.

Reply:
@kwanhur 
We can use this: https://github.com/apache/apisix/blob/51eec3ea64bba464ab4a6741acc318c502cab26c/apisix/schema_def.lua#L86

Reply:
Thx, Done.